android/java/strings/android_brave_strings.grd

```diff
@@ -102,6 +102,9 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_ACCESSIBILITY_TOOLBAR_BTN_BRAVE_REWARDS" desc="Content description for the button that is enable, disable Brave Rewards.">
         Manage Brave Rewards
       </message>
+      <message name="IDS_PREFS_ABOUT_CHROME" desc="Title for the About Chrome page. [CHAR-LIMIT=32]">
```

Comment:
I think it makes sense to keep this message here temporarily and create a new PR to handle l10n translation of the `android_chrome_strings.grd` file. I looked into it briefly and there are some complications (mainly involved around suppressing Android lint warnings, etc.)

Reply:
Yes, right here: https://github.com/brave/brave-browser/issues/5908